media_kit/lib/src/player/native/player/real.dart

```diff
@@ -2752,10 +2758,11 @@ void _seek(_SeekData data) {
   final mpv = generated.MPV(DynamicLibrary.open(data.lib));
   final ctx = Pointer<generated.mpv_handle>.fromAddress(data.ctx);
   // ---------
-  final duration = data.duration;
+  final duration = data.seekDuration - data.videoPosition;
   // ---------
   final value = duration.inMilliseconds / 1000;
-  final command = 'seek ${value.toStringAsFixed(4)} absolute'.toNativeUtf8();
+
+  final command = 'seek ${value.toStringAsFixed(4)}'.toNativeUtf8();
```

Comment:
By default, mpv does `seek relative+keyframes <number>`. The tests are failing because seeking is no longer `absolute`. You can do the following:
```dart
final command = 'seek ${value.toStringAsFixed(4)} absolute+keyframes'.toNativeUtf8();
```

I don't think this should be a default though, but it might be worth adding a new player configuration to either use accurate but expensive or inaccurate but fast seeking method.

Reply:
I did try this configuration it fails very badly if you seek multiple times.

And the default for mpv from the documentation is `relative` from what I found 


For the configuration part I agree that it can be optional, but I don't have any idea how to expose the option cleanly